README.md

```diff
@@ -151,6 +151,7 @@ The above command pulled the existing data out of Route53 and placed the results
 |--|--|--|--|
 | [AzureProvider](/octodns/provider/azuredns.py) | A, AAAA, CNAME, MX, NS, PTR, SRV, TXT | No | |
 | [CloudflareProvider](/octodns/provider/cloudflare.py) | A, AAAA, CAA, CNAME, MX, NS, SPF, TXT | No | CAA tags restricted |
+| [DigitalOceanProvider](/octodns/provider/digitalocean.py) | A, AAAA, CAA, CNAME, MX, NS, TXT, SRV | No | |
```

Comment:
Not sure what the CAA tags restricted means so I didn't put it.

Reply:
I believe that was supposed to mean that they support the record type, but have restrictions on the values beyond what the spec lays out. The spec on CAA lays out a way to add non-standard/extra values and to flip a flag saying that if a certificate authority doesn't understand them it shouldn't issue certs for the zone. I believe these messages are saying the providers don't support all of that, they just support the base/initial set of laid out value types.